GANDLF/cli/deploy.py

```diff
@@ -18,30 +18,30 @@
 
 
 def run_deployment(
-    mlcubedir,
-    outputdir,
-    target,
-    mlcube_type,
-    entrypoint_script=None,
-    configfile=None,
-    modeldir=None,
-    requires_gpu=None,
-):
+    mlcubedir: str,
+    outputdir: str,
+    target: str,
+    mlcube_type: str,
```

Comment:
In the cases when the param is typed to string and default set to None, as in the previous cases it should be Union[str,None]

Reply:
Fixed with `Optional`.